libsolidity/formal/SymbolicVariables.cpp

```diff
@@ -197,92 +209,111 @@ void SymbolicFunctionVariable::resetDeclaration()
 	m_declaration = m_context.newVariable(currentName(), m_sort);
 }
 
-SymbolicMappingVariable::SymbolicMappingVariable(
+SymbolicEnumVariable::SymbolicEnumVariable(
```

Comment:
This diff is just mean :-D.

Reply:
Yea, I always select `hide whitespace changes` :p

Reply:
Doesn't help for this one, though :-).